stdlib/public/core/StringStorage.swift

```diff
@@ -258,14 +308,20 @@ extension _StringStorage {
   }
 
   @nonobjc
-  private var isASCII: Bool { return _countAndFlags.isASCII }
+  private var isASCII: Bool {
+#if arch(i386) || arch(arm)
+    return _flags.isASCII
+#else
+    return _countAndFlags.isASCII
+#endif
+  }
 
   @nonobjc
   // @opaque
   internal var _breadcrumbsAddress: UnsafeMutablePointer<_StringBreadcrumbs?> {
```

Comment:
Were you able to test that bread crumbing was triggering on 32-bit?

Reply:
Yeah, it seems to work. I looked at some UTF16View operations; populateBreadcrumbs got called as expected.